network/epcommon/endpoint_common.go

```diff
@@ -35,22 +38,23 @@ const (
 	acceptRAV6File       = "/proc/sys/net/ipv6/conf/%s/accept_ra"
 )
 
-func getPrivateIPSpace() []string {
-	privateIPAddresses := []string{"10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16", "169.254.0.0/16"}
-	return privateIPAddresses
+var errorEPCommon = errors.New("ErrorEPCommon Error")
+
+func newErrorEPCommon(errStr string) error {
+	return fmt.Errorf("ErrorEPCommon %w : %s", errorEPCommon, errStr)
 }
 
-func getFilterChains() []string {
-	chains := []string{"FORWARD", "INPUT", "OUTPUT"}
-	return chains
+type EPCommon struct {
```

Comment:
What is the meaning of `Common` in this context? Seems like this file is specific to linux. What is this common to?

Reply:
We decided to rename the file to networkutility_linux but will be part of a 2nd PR with other changes. Because i am blocking some changes from Shriroop and Ali.